static/app/utils/replays/hooks/useExtractedCrumbHtml.tsx

```diff
@@ -0,0 +1,120 @@
+import {useEffect, useState} from 'react';
+import first from 'lodash/first';
+import {Replayer} from 'rrweb'; // , ReplayerEvents
+import {eventWithTime} from 'rrweb/typings/types';
+
+import type {Crumb} from 'sentry/types/breadcrumbs';
+import type ReplayReader from 'sentry/utils/replays/replayReader';
+
+const IncrementalSnapshot = 3;
```

Comment:
You could use the enum https://github.com/rrweb-io/rrweb/blob/master/packages/rrweb/src/types.ts#L18

Reply:
I can't sort out how to import it. both `import {EventType} from 'rrweb/typings';` and `import {EventType} from 'rrweb/typings/types';` fail like this:
<img width="600" alt="Screen Shot 2022-07-21 at 2 06 09 PM" src="https://user-images.githubusercontent.com/187460/180315513-5f52262f-6202-458c-979a-fd792d4eff47.png">
 
the `import {eventWithTime} ...` does work, but that's a TS only import, not webpack.

So I've copied in the whole enum and commented its origin. Updating to rrweb 2.0 might be fun here.